meta.go

```diff
@@ -0,0 +1,49 @@
+package spotify
+
+import (
+	"net/url"
+	"strings"
+)
+
+type HREF struct {
+	*url.URL
+}
+
+func (h *HREF) Get(api *API, obj interface{}) error {
+	path := strings.Replace(h.Path, "/v1", "", 1)
+	return api.get(path, obj)
+}
+
+func (h *HREF) UnmarshalJSON(data []byte) error {
+	u, err := url.Parse(string(data[1 : len(data)-1]))
+	if err != nil {
+		return err
+	}
+	h.URL = u
+	return nil
+}
+
+type Meta struct {
+	HREF         *HREF
```

Comment:
In the docs, this is listed as a `string`. I think it would be good practice to keep it that way, but we could write a `GetURL() *url.URL` function for anyone who might benefit from it.

Reply:
I could do that, yeah. I was just being a bit slick here.